uint8 position
uint8 speed
uint8 force
uint8 command
uint8 stop
